script.js

```diff
@@ -0,0 +1,121 @@
+
+var canvas = document.getElementById("canvas");
+var ctx = canvas.getContext("2d");
+var radius = canvas.height / 2;
+ctx.translate(radius, radius);
+radius = radius * 0.90
+setInterval(drawClock, 1000) //Analog Clock
+currentTime();  // Digital Clock
+
+// Drawing the Clock;s canvas and face
+function drawClock() {
+    drawFace(ctx, radius);
+}
+
+function drawFace(ctx, radius) {
+    var grad;
+
+    ctx.beginPath();
+    ctx.arc(0, 0, radius, 0, 2 * Math.PI);
+    ctx.fillStyle = "white";
+    ctx.fill();
+
+    grad = ctx.createRadialGradient(0, 0, radius * 0.95, 0, 0, radius * 1.05);
+    grad.addColorStop(0, '#333');
+    grad.addColorStop(0.5, 'white');
+    grad.addColorStop(1, '#333');
+    ctx.strokeStyle = grad;
+    ctx.lineWidth = radius * 0.1;
+    ctx.stroke();
+    
+    ctx.beginPath();
+    ctx.arc(0, 0, radius * 0.1, 0 , 2 * Math.PI);
+    ctx.fillStyle = '#333';
+    ctx.fill();
+}
+
+// Drawing the clock's numbers
+
+function drawClock() {
+    drawFace(ctx, radius);
+    drawNumbers(ctx, radius);
+    drawTime(ctx, radius);
+  }
+  function drawNumbers(ctx, radius) {
+    var ang;
+    var num;
+    ctx.font = radius * 0.15 + "px arial";
+    ctx.textBaseline = "middle";
+    ctx.textAlign = "center";
+    for(num = 1; num < 13; num++){

```

Comment:
13 --> make it a CONST, also 12 and a <= makes it more human readable since I assume this is numbers on a clock